src/olympia/devhub/templates/devhub/addons/onboarding_subscription.html

```diff
@@ -0,0 +1,79 @@
+{% extends "devhub/base_impala.html" %}
+
+{% set title = _('Set Up Payment') %}
+
+{% block js %}
+{{ super() }}
+<script src="https://js.stripe.com/v3/"></script>
```

Comment:
Just wondering why this is here, rather than in our `stripe.js`?

Reply:
this requires a js file from stripe

Reply:
And stripe themselves ask you specifically require their js from their site rather than copying it into your bundle